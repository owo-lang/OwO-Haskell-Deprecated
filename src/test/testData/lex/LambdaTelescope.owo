\x -> y
(x : a -> b) -> c
{a : C} -> D
{| x : y |} -> c
\x -> \y -> z
