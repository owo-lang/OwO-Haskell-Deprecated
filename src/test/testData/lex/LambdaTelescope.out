BackslashToken (0 1 1) (1 1 2)
Identifier x (1 1 2) (2 1 3)
RightArrowToken (3 1 4) (5 1 6)
Identifier y (6 1 7) (7 1 8)
ParenthesisLToken (8 2 1) (9 2 2)
Identifier x (9 2 2) (10 2 3)
ColonToken (11 2 4) (12 2 5)
Identifier a (13 2 6) (14 2 7)
RightArrowToken (15 2 8) (17 2 10)
Identifier b (18 2 11) (19 2 12)
ParenthesisRToken (19 2 12) (20 2 13)
RightArrowToken (21 2 14) (23 2 16)
Identifier c (24 2 17) (25 2 18)
BraceLToken (26 3 1) (27 3 2)
Identifier a (27 3 2) (28 3 3)
ColonToken (29 3 4) (30 3 5)
Identifier C (31 3 6) (32 3 7)
BraceRToken (32 3 7) (33 3 8)
RightArrowToken (34 3 9) (36 3 11)
Identifier D (37 3 12) (38 3 13)
InstanceArgumentLToken (39 4 1) (41 4 3)
Identifier x (42 4 4) (43 4 5)
ColonToken (44 4 6) (45 4 7)
Identifier y (46 4 8) (47 4 9)
InstanceArgumentRToken (48 4 10) (50 4 12)
RightArrowToken (51 4 13) (53 4 15)
Identifier c (54 4 16) (55 4 17)
BackslashToken (56 5 1) (57 5 2)
Identifier x (57 5 2) (58 5 3)
RightArrowToken (59 5 4) (61 5 6)
BackslashToken (62 5 7) (63 5 8)
Identifier y (63 5 8) (64 5 9)
RightArrowToken (65 5 10) (67 5 12)
Identifier z (68 5 13) (69 5 14)
